app/controllers/profiles_controller.rb

```diff
@@ -1,4 +1,11 @@
 class ProfilesController < ApplicationController
+  before_action :user_signed_in, only: :show
+  before_action :correct_user, only: :show
```

Comment:
you can do it in one line if you want. 

`before_action : user_signed_in, : correct_user, only: :show`